### PR TITLE
Task-47220: Inconsistence of WYSIWYG in Table display

### PR DESCRIPTION
**Pull request by lbengayed**
When a table does have a long text, some words are cut and the line break is in the middle of the words.
This PR fix it

### CODE REVIEW

File: notes-service/src/main/resources/locale/portlet/Portlets_en.properties [hunk 1]
```diff
@@ -0,0 +1,2 @@
+SpaceSettings.application.notes.application.title=Notes Application
```
Comment (ahamdi) [inline notes-service/src/main/resources/locale/portlet/Portlets_en.properties]: this file should be removed

File: notes-webapp/src/main/webapp/skin/less/notes/notes.less [hunk 2]
```diff
@@ -153,11 +153,19 @@
   .notes-wrapper {
     min-height: calc(~"100vh - 170px");
   }
+  .notes-application-content{
+	overflow: auto;
```
Comment (ahamdi) [inline notes-webapp/src/main/webapp/skin/less/notes/notes.less]: Fix indentation of this file

File: notes-webapp/src/main/webapp/skin/less/notes/notes.less [hunk 3]
```diff
@@ -153,11 +153,19 @@
...
+  }
   table{
-	table-layout: fixed;
-	width: 100%;
-	word-break: break-all;
-	border: 1px solid var(--allPagesBorderColor,#e1e8ee);
+	  border: 1px solid var(--allPagesBorderColor,#e1e8ee);
+	  border-right: none;
```
Comment (ahamdi) [inline notes-webapp/src/main/webapp/skin/less/notes/notes.less]: Consider RTLisation please

File: notes-webapp/src/main/webapp/skin/less/notes/notes.less [hunk 4]
```diff
@@ -153,11 +153,19 @@
...
+	  border-top: none;
+	  width: auto;
+	  td {
+		  border-right: 1px solid var(--allPagesBorderColor,#e1e8ee);
```
Comment (ahamdi) [inline notes-webapp/src/main/webapp/skin/less/notes/notes.less]: Consider RTLisation please

**Review by ahamdi — CHANGES_REQUESTED** [review]
Please add a description for this PR

**Review by rdenarie — CHANGES_REQUESTED** [review]
Hello,
Can you add a description following [our commit message best practises](https://developer.exoplatform.org/#id-commits-messages-best-practices)
It help us to understand what you want to do

**Review by rdenarie — APPROVED** [review]

**Review by ahamdi — APPROVED** [review]

**Pull request merged**